charts/access/email/templates/configmap.yaml

```diff
@@ -0,0 +1,35 @@
+apiVersion: v1
+kind: ConfigMap
+metadata:
+  name: {{ include "email.fullname" . }}
+  labels:
+    {{- include "email.labels" . | nindent 4 }}
+data:
+  teleport-email.toml: |
+    [teleport]
```

Comment:
Seems to be some missing config options in this section. See https://github.com/gravitational/teleport-plugins/blob/9e149895f54edd4870f760be08a4c97dae49c151/lib/config.go#L18

Reply:
`AuthServer` deprecates `Addr` and the fields `ClientKey`, `ClientCrt` and `RootCAs` are just another way to pass the same data that's supposed to be in the file pointed by `Identity`. @walt and I agreed that we should only support "best practices" and avoid multiple ways to configure the same thing. That's why I've left out those fields after investigating what they are actually doing.